### PR TITLE
add helper/wrapper to list all tables in DB

### PR DESCRIPTION
**Pull request by iveksl2**

### CODE REVIEW

**Comment by iveksl2** [comment]
failing CI, dont merge

File: R/run_query.R [hunk 1]
```diff
@@ -17,3 +17,13 @@ run_query <- function(query, conn = last_connection()) {
   stopifnot(is.character(query))
   DBI::dbGetQuery(conn, query)
 }
+
+#' Lists all tables within DB 
+#'
+#' @export
+all_tables <- function() {
+  berdie::run_query("select table_name
```
Comment (robertzk) [inline R/run_query.R]: Can you remove `berdie::`? We don't need to, since we're in the package. :)

**Comment by robertzk** [comment]
Can you redocument? Somehow the `@export` didn't take.

**Comment by robertzk** [comment]
Cool helper! Let's get it merged after CI.

File: R/run_query.R (hunk 1 above)
Comment (kirillseva) [inline R/run_query.R]: Please just use `DBI::dbListTables(dbconn)`

**Comment by kirillseva** [comment]
Please address my comment and let's merge

**Comment by robertzk** [comment]
Can you add a trivial test and then generate the Rmd for the markdown?

File: tests/testthat.R [hunk 2]
```diff
@@ -0,0 +1,3 @@
+library(testthat)
+library(berdie)
+test_package("berdie")
```
Comment (kirillseva) [inline tests/testthat.R]: pretty sure it has to be test_check.

This is what my `devtools` tells me

Comment (robertzk) [inline tests/testthat.R]: Ah, correct.

**Pull request closed without merging**